src/Aquifer.Jobs/SendNewContentEmails.cs

```diff
@@ -0,0 +1,100 @@
+using Aquifer.Common;
+using Aquifer.Common.Clients;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Aquifer.Jobs.Configuration;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using SendGrid.Helpers.Mail;
+
+namespace Aquifer.Jobs;
+
+public class SendNewContentEmails(AquiferDbContext dbContext, SendGridClient client, IOptions<ConfigurationOptions> options)
+{
+    [Function(nameof(SendNewContentEmails))]
+    public async Task Run([TimerTrigger("%NewContentEmail:CronSchedule%")] TimerInfo timerInfo, CancellationToken ct)
+    {
```

Comment:
This method feels too big to me. I'd generally like to see this split into some private methods like `GetSubscribers` and `GetAllNewItems` etc. so that we're following SRP better. It helps with readability and maintainability.